src/sonic-config-engine/tests/sample-voq-graph.xml

```diff
@@ -79,6 +79,20 @@
         </a:Properties>
         <a:Key>linecard-1:Ethernet1/1;ARISTA01-RH:Ethernet1/1</a:Key>
       </a:LinkMetadata>
+      <a:LinkMetadata>
+        <a:Name i:nil="true"/>
+        <a:Properties>
+          <a:DeviceProperty>
+            <a:Name>Frequency</a:Name>
+            <a:Value>195.875</a:Value>
+          </a:DeviceProperty>
+          <a:DeviceProperty>
+            <a:Name>TxPower</a:Name>
+            <a:Value>-10</a:Value>
```

Comment:
can we have one test case for float value? like -10.3? see https://github.com/sonic-net/sonic-buildimage/pull/11053

Reply:
updated the test to test this